modules/adformOpenRTBBidAdapter.js

```diff
@@ -66,11 +66,28 @@ export const spec = {
         };
         if (props) {
           asset.id = props.id;
+          let wmin, hmin, w, h;
+          let aRatios = bidParams.aspect_ratios;
+
+          if (aRatios) {
```

Comment:
I'd suggest adding `&& aRatios[0]` to check that the 1st element is also defined